services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/TransactWriteItemsEnhancedRequest.java

```diff
@@ -67,7 +67,45 @@ public static Builder builder() {
     }
 
     /**
-     * Gets the ClientRequestToken of this enhanced request.
+     * <p>
+     * Providing a <code>ClientRequestToken</code> makes the call to <code>TransactWriteItems</code> idempotent, meaning
+     * that multiple identical calls have the same effect as one single call.
+     * </p>
+     * <p>
+     * Although multiple identical calls using the same client request token produce the same result on the server (no
+     * side effects), the responses to the calls might not be the same. If the <code>ReturnConsumedCapacity&gt;</code>
+     * parameter is set, then the initial <code>TransactWriteItems</code> call returns the amount of write capacity
+     * units consumed in making the changes. Subsequent <code>TransactWriteItems</code> calls with the same client token
+     * return the number of read capacity units consumed in reading the item.
+     * </p>
+     * <p>
+     * A client request token is valid for 10 minutes after the first request that uses it is completed. After 10
+     * minutes, any request with the same client token is treated as a new request. Do not resubmit the same request
+     * with the same client token for more than 10 minutes, or the result might not be idempotent.
+     * </p>
+     * <p>
+     * If you submit a request with the same client token but a change in other parameters within the 10-minute
+     * idempotency window, DynamoDB returns an <code>IdempotentParameterMismatch</code> exception.
+     * </p>
+     *
+     * @return Providing a <code>ClientRequestToken</code> makes the call to <code>TransactWriteItems</code> idempotent,
```

Comment:
Why paste it twice?

Reply:
It's already been there in the original javadoc. I was also confused why it's there twice but I thought it could be some unkown specific standard so I kept it...